src/main/scala/com/jcdecaux/setl/storage/connector/JDBCConnector.scala

```diff
@@ -71,8 +71,8 @@ class JDBCConnector(val conf: JDBCConnectorConf) extends DBConnector {
     log.warn("Create is not supported in JDBC Connector")
   }
 
-  override def delete(query: String): Unit = {
-    this.executeRequest(query)
+  override def delete(condition: String): Unit = {
+    this.executeRequest(s"DELETE FROM ${conf.getDbTable.get} WHERE $condition;")
```

Comment:
check if the condition contains `;`. because somebody can give a condition like `x = 'some_value'; drop table table_name`.

throw an exception if true